examples/tutorial/03_Alignment_and_Preprocessing.ipynb

```diff
@@ -4,21 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Alignment and Preprocessing"
```

Comment:
This split is intentional for website usage. It allows the notebook to render with the `nbsite` generated title on the website, and not have it duplicated in the content.